makedmdconf.d

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env rdmd
+module makedmdconf;
+
+import core.stdc.stdlib : exit;
+import std.getopt, std.string;
+import std.path, std.file, std.stdio;
+import std.process;
+
+int main(string[] args)
+{
+    bool mscoff = false;
+
+    auto helpInfo = getopt(args,
+        "mscoff", &mscoff);
+    enum nonOptionArgCount = 3;
+    const wrongArgCount = (args.length != nonOptionArgCount + 1);
+    if (wrongArgCount || helpInfo.helpWanted)
+    {
+        if (wrongArgCount)
+            writefln("Error: expected %s non-option args but got %s", nonOptionArgCount, args.length - 1);
+        defaultGetoptPrinter("Usage: makedmdconf [-options] <outfile> <os> <bulid>", helpInfo.options);
+        return 1;
+    }
+
+    const outFile = args[1];
+    const os      = args[2];
+    const build   = args[3];
+
+    if (updateIfChanged(outFile, generateDmdConf(os, build, mscoff)))
+        writefln("updated '%s'", outFile);
+    else
+        writefln("already up-to-date '%s'", outFile);
+    return 0;
+}
+
+string generateDmdConf(string os, string build, bool mscoff)
+{
+    string sharedflags = "";
+    string model32flags = "";
+    string model64flags = "";
+
+    sharedflags  ~= " -I%@P%/../../../../../druntime/import";
+    sharedflags  ~= " -I%@P%/../../../../../phobos";
+    model32flags ~= " -L-L%@P%/../../../../../phobos/generated/{os}/{build}/32";
+    model64flags ~= " -L-L%@P%/../../../../../phobos/generated/{os}/{build}/64";
+
+    if (os == "windows")
+    {
+        // NOTE: I don't think I need to add user32/kernel32 because I think the
```

Comment:
1) Not a good style to use the first person in remarks as no one knows immediately who you are
2) Let's better verify before we add such comments. Currently, the sc.ini isn't generated for Windows, but the release sc.ini doesn't look like it needs this: https://github.com/dlang/dmd/blob/master/ini/windows/bin/sc.ini

Reply:
Yeah the plan isn't to merge this comment, it's to let me and reviewers know it's something that needs to be addressed before merging.